src/components/AddPaymentMethodMenu.tsx

```diff
@@ -74,14 +75,24 @@ function AddPaymentMethodMenu({
 
     const isPersonalOnlyOption = canUsePersonalBankAccount && !canUseBusinessBankAccount;
 
+    const onPaymentItemSelected = useCallback(
```

Comment:
Opssss. Why do we need to change this file? AddPaymentMethodMenu is only used in WalletPage. Did I miss anything?

Reply:
I was just following the frontend changes directions from the design doc, but I agree on this one, which is why I've added the `completePaymentOnboarding` method to other places as well. I'll remove it from `AddPaymentMethodMenu`